samples/ControlCatalog/Pages/ClipboardPage.xaml.cs

```diff
@@ -25,7 +25,7 @@ private void InitializeComponent()
 
         private async void CopyText(object sender, RoutedEventArgs args)
         {
-            await Application.Current.Clipboard.SetTextAsync(ClipboardContent.Text);
+            await Application.Current?.Clipboard?.SetTextAsync(ClipboardContent?.Text);
```

Comment:
As for me it should create another warning. Since you can't await a null object.

Reply:
restored, I will fixing in other PR